src/main/java/app/model/hbm/Order.hbm.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+"http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<!-- Generated Jun 19, 2019 8:07:42 AM by Hibernate Tools 3.5.0.Final -->
+<hibernate-mapping>
+    <class name="app.model.Order" table="ORDER">
+        <id name="id" type="java.lang.Integer">
+            <column name="id" />
+            <generator class="assigned" />
+        </id>
+        <property name="nameReceiver" type="java.lang.String">
+            <column name="receiver" />
+        </property>
+        <property name="address" type="java.lang.String">
+            <column name="address" />
+        </property>
+        <property name="phone" type="java.lang.String">
+            <column name="phone" />
+        </property>
+        <property name="total" type="java.lang.Float">
+            <column name="total" />
+        </property>
+        <property name="note" type="java.lang.String">
+            <column name="note" />
+        </property>
+        <property name="status" type="java.lang.Integer">
+            <column name="status" />
+        </property>
+        <many-to-one name="user" class="app.model.User" fetch="join">
```

Comment:
` fetch="join"` xóa dc hơm chú

Reply:
dạ chắc được, nó sẽ dùng cái setting default ạ. many-to-one chắc là select, em nghĩ rứa :(